substrate/client/transaction-pool/Cargo.toml

```diff
@@ -51,7 +52,16 @@ sp-consensus = { workspace = true, default-features = true }
 substrate-test-runtime = { workspace = true }
 substrate-test-runtime-client = { workspace = true }
 substrate-test-runtime-transaction-pool = { workspace = true }
+thiserror = { workspace = true }
+tokio = { workspace = true, features = ["rt-multi-thread"] }
+tracing-subscriber = { workspace = true }
+txtesttool = { git = "https://github.com/iulianbarbu/tx-test-tool.git", branch = "provide-a-ttxt-api" }
```

Comment:
Please try to use released version: https://crates.io/crates/substrate-txtesttool